### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.management.compute._inner_error
    id: _inner_error
    parent: com.microsoft.azure.management.compute
    children:
      - com.microsoft.azure.management.compute._inner_error.errordetail()
      - com.microsoft.azure.management.compute._inner_error.exceptiontype()
      - com.microsoft.azure.management.compute._inner_error.withErrordetail(String)
      - com.microsoft.azure.management.compute._inner_error.withExceptiontype(String)
    href: com.microsoft.azure.management.compute._inner_error.yml
    langs:
      - java
    name: InnerError
    nameWithType: InnerError
    fullName: com.microsoft.azure.management.compute.InnerError
    type: Class
    source:
      remote: &ref_0
        path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
        branch: v1.3.0
        repo: 'https://github.com/Azure/azure-sdk-for-java'
      path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
      startLine: 15
    package: com.microsoft.azure.management.compute
    summary: <p>Interner Fehler-Details. </p>
    syntax: &ref_1
      content: public class InnerError
    inheritance:
      - java.lang.Object
  - uid: com.microsoft.azure.management.compute._inner_error.errordetail()
    id: errordetail()
    parent: com.microsoft.azure.management.compute._inner_error
    href: com.microsoft.azure.management.compute._inner_error.yml
    langs:
      - java
    name: errordetail()
    nameWithType: InnerError.errordetail()
    fullName: String com.microsoft.azure.management.compute.InnerError.errordetail()
    overload: com.microsoft.azure.management.compute._inner_error.errordetail*
    type: Method
    source:
      remote: *ref_0
      path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
      startLine: 53
    package: com.microsoft.azure.management.compute
    summary: |-
      <p>Ruft den Wert "errordetail".</p>
      <p></p>
    syntax:
      content: public String errordetail()
      return:
        type: '26831127'
        description: <p>der Wert "errordetail" </p>
  - uid: com.microsoft.azure.management.compute._inner_error.exceptiontype()
    id: exceptiontype()
    parent: com.microsoft.azure.management.compute._inner_error
    href: com.microsoft.azure.management.compute._inner_error.yml
    langs:
      - java
    name: exceptiontype()
    nameWithType: InnerError.exceptiontype()
    fullName: String com.microsoft.azure.management.compute.InnerError.exceptiontype()
    overload: com.microsoft.azure.management.compute._inner_error.exceptiontype*
    type: Method
    source:
      remote: *ref_0
      path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
      startLine: 33
    package: com.microsoft.azure.management.compute
    summary: |-
      <p>Ruft den Wert exceptionType bereit.</p>
      <p></p>
    syntax:
      content: public String exceptiontype()
      return:
        type: '26831127'
        description: <p>der Wert exceptionType bereit </p>
  - uid: com.microsoft.azure.management.compute._inner_error.withErrordetail(String)
    id: withErrordetail(String)
    parent: com.microsoft.azure.management.compute._inner_error
    href: com.microsoft.azure.management.compute._inner_error.yml
    langs:
      - java
    name: withErrordetail(String errordetail)
    nameWithType: InnerError.withErrordetail(String errordetail)
    fullName: InnerError com.microsoft.azure.management.compute.InnerError.withErrordetail(String errordetail)
    overload: com.microsoft.azure.management.compute._inner_error.withErrordetail*
    type: Method
    source:
      remote: *ref_0
      path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
      startLine: 63
    package: com.microsoft.azure.management.compute
    summary: |-
      <p>Legen Sie den Wert "errordetail".</p>
      <p></p>
    syntax:
      content: public InnerError withErrordetail(String errordetail)
      parameters:
        - id: errordetail
          type: '26831127'
          description: <p>Der festzulegende Wert "errordetail" </p>
      return:
        type: com.microsoft.azure.management.compute._inner_error
        description: <p>die <xref uid="com.microsoft.azure.management.compute._inner_error" data-throw-if-not-resolved="false">InnerError</xref> Objekt selbst. </p>
  - uid: com.microsoft.azure.management.compute._inner_error.withExceptiontype(String)
    id: withExceptiontype(String)
    parent: com.microsoft.azure.management.compute._inner_error
    href: com.microsoft.azure.management.compute._inner_error.yml
    langs:
      - java
    name: withExceptiontype(String exceptiontype)
    nameWithType: InnerError.withExceptiontype(String exceptiontype)
    fullName: InnerError com.microsoft.azure.management.compute.InnerError.withExceptiontype(String exceptiontype)
    overload: com.microsoft.azure.management.compute._inner_error.withExceptiontype*
    type: Method
    source:
      remote: *ref_0
      path: azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/InnerError.java
      startLine: 43
    package: com.microsoft.azure.management.compute
    summary: |-
      <p>Legen Sie den Exceptiontype-Wert.</p>
      <p></p>
    syntax:
      content: public InnerError withExceptiontype(String exceptiontype)
      parameters:
        - id: exceptiontype
          type: '26831127'
          description: <p>Der festzulegende Wert exceptionType bereit </p>
      return:
        type: com.microsoft.azure.management.compute._inner_error
        description: <p>die <xref uid="com.microsoft.azure.management.compute._inner_error" data-throw-if-not-resolved="false">InnerError</xref> Objekt selbst. </p>
references:
  - uid: '26831127'
    spec.java:
      - name: String
        fullName: String
  - uid: com.microsoft.azure.management.compute._inner_error.exceptiontype*
    name: exceptiontype
    nameWithType: InnerError.exceptiontype
    fullName: String com.microsoft.azure.management.compute.InnerError.exceptiontype
    package: com.microsoft.azure.management.compute
  - uid: com.microsoft.azure.management.compute._inner_error.withExceptiontype*
    name: withExceptiontype
    nameWithType: InnerError.withExceptiontype
    fullName: InnerError com.microsoft.azure.management.compute.InnerError.withExceptiontype
    package: com.microsoft.azure.management.compute
  - uid: com.microsoft.azure.management.compute._inner_error.errordetail*
    name: errordetail
    nameWithType: InnerError.errordetail
    fullName: String com.microsoft.azure.management.compute.InnerError.errordetail
    package: com.microsoft.azure.management.compute
  - uid: com.microsoft.azure.management.compute._inner_error.withErrordetail*
    name: withErrordetail
    nameWithType: InnerError.withErrordetail
    fullName: InnerError com.microsoft.azure.management.compute.InnerError.withErrordetail
    package: com.microsoft.azure.management.compute
  - uid: com.microsoft.azure.management.compute._inner_error
    parent: com.microsoft.azure.management.compute
    href: com.microsoft.azure.management.compute._inner_error.yml
    name: InnerError
    nameWithType: InnerError
    fullName: com.microsoft.azure.management.compute.InnerError
    type: Class
    summary: <p>Interner Fehler-Details. </p>
    syntax: *ref_1